components/backup/src/utils.rs

```diff
@@ -90,11 +90,11 @@ pub fn create_tokio_runtime(thread_count: usize, thread_name: &str) -> TokioResu
         .thread_name(thread_name)
         .enable_io()
         .enable_time()
-        .on_thread_start(|| {
+        .after_start_wrapper(|| {
```

Comment:
minor: IMO, on_thread_start_wrapper/on_thread_stop_wrapper seem a better naming.

Reply:
`after_start_wrapper` and `before_stop_wrapper` comes from `ThreadBuildWrapper` trait. I implement this trait for different kinds of thread pool builder, so their wrapper function names are same.